test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -808,6 +820,378 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
+				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+
+				//Update the profile to disable HighPowerConsumption and enable PerPodPowerManagment
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption:  pointer.Bool(true),
+					RealTime:              pointer.Bool(true),
+					PerPodPowerManagement: pointer.BoolPtr(false),
+				}
+				if !*profile.Spec.RealTimeKernel.Enabled {
+					profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
+						Enabled: pointer.BoolPtr(true),
+					}
+				}
+
+				By("Patching the performance profile with workload hints")
+				newspec, err := json.Marshal(profile.Spec)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, newspec)),
+					),
+				)).ToNot(HaveOccurred())
+
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+
+				By("Waiting when mcp finishes updates")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+
+				stalldEnabled, rtKernel = true, true
+				noHzParam = fmt.Sprintf("nohz_full=%s", *profile.Spec.CPU.Isolated)
+				sysctlMap = map[string]string{
+					"kernel.hung_task_timeout_secs": "600",
+					"kernel.nmi_watchdog":           "0",
+					"kernel.sched_rt_runtime_us":    "-1",
+					"vm.stat_interval":              "10",
+				}
+
+				kernelParameters = []string{noHzParam, "tsc=nowatchdog", "nosoftlockup", "nmi_watchdog=0", "mce=off", "skew_tick=1",
+					"processor.max_cstate=1", "intel_idle.max_cstate=0", "intel_pstate=disable", "idle=poll"}
+				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+			})
+
+			It("[test_id:54184]Verify enabling both HighPowerConsumption and PerPodPowerManagment fails", func() {
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					PerPodPowerManagement: pointer.BoolPtr(true),
+					HighPowerConsumption:  pointer.BoolPtr(true),
+					RealTime:              pointer.BoolPtr(true),
+				}
+				EventuallyWithOffset(1, func() string {
+					err := testclient.Client.Update(context.TODO(), profile)
+					if err != nil {
+						statusErr, _ := err.(*errors.StatusError)
+						return statusErr.Status().Message
+					}
+					return fmt.Sprint("Profile applied successfully")
+				}, time.Minute, 5*time.Second).Should(ContainSubstring("HighPowerConsumption and PerPodPowerManagement can not be both enabled"))
+			})
+
+			It("[test_id:54185] Verify sysfs paramters of guaranteed pod with powersave annotiations", func() {
```

Comment:
nit: spelling - parameters and annotations

Reply:
@bartwensley  addressed this in the latest patch